packages/core/src/components/search/SearchInput/SearchInput.tsx

```diff
@@ -21,13 +21,13 @@ import {
   useState,
 } from 'react'
 
```

Comment:
suggestion: Add a TODO comment on this file to remind us that we should delete it once the migration is finished.

Reply:
nice. Done [here](https://github.com/vtex/faststore/pull/2347/commits/9eea6a3e3fd610ae96242917e5a1ad86cab8fe92).